pom.xml

```diff
@@ -124,6 +124,9 @@
         <maven-shade-plugin.version>3.2.4</maven-shade-plugin.version>
         <maven-helper-plugin.version>3.2.0</maven-helper-plugin.version>
         <influxdb-java.version>2.22</influxdb-java.version>
+        <jackson.version>2.10.5</jackson.version>
+        <slf4j.api.version>1.7.5</slf4j.api.version>
+        <slf4j.log4j12.version>1.7.5</slf4j.log4j12.version>
```

Comment:
Why not use the latest stable version?

Reply:
which version?

Reply:
> which version?

The `slf4j` version seems a bit old, the current version was released in `2013`